client/client.go

```diff
@@ -21,6 +21,7 @@ type Client struct {
 	engine              engine.Engine // The core business logic of the client
 	Address             *types.Address
 	completedObjectives chan protocols.ObjectiveId
+	store               *store.Store
```

Comment:
What's the thinking here? Currently, the engine owns the store and the client can't access it directly.

Reply:
Currently the `store` is used by the `client` to find the appropriate ledger channels to construct a new VirtualFund objective.